1-js/05-data-types/07-map-set/article.md

```diff
@@ -39,57 +39,57 @@ alert( map.get('1') ); // 'str1'
 alert( map.size ); // 3
 ```
 
-As we can see, unlike objects, keys are not converted to strings. Any type of key is possible.
+এখানে আমরা দেখছি এটি কী কে অবজেক্টের মত স্ট্রিংয়ে রূপান্তর করে না। সুতরাং যেকোন ডাটা টাইপের কী রাখা সম্ভব।
 
-```smart header="`map[key]` isn't the right way to use a `Map`"
-Although `map[key]` also works, e.g. we can set `map[key] = 2`, this is treating `map` as a plain JavaScript object, so it implies all corresponding limitations (no object keys and so on).
+```smart header=" `Map` এর মধ্যে `map[key]` ব্যবহার করা উচিত নই"
+যদিও `map[key]` এটি কাজ করে, যেমন আমরা `map[key] = 2` সেট করতে পারি, এক্ষেত্রে ইঞ্জিন `map` কে  plain JavaScript object হিসেবে ধরে নেয়, সুতরাং এভাবে করা উচিত নই।
```

Comment:
missing some information from the original.

Reply:
এর ভাল কোন অনুবাদ না পেয়ে এটা লিখেছি  _সুতরাং এভাবে করা উচিত নই।_

Reply:
`নই` এর যায়গায় `নয়` লিখুন।